consensus/tests/consensus_proxy.rs

```diff
@@ -17,11 +17,10 @@ use nimiq_test_utils::{
     node::TESTING_BLS_CACHE_MAX_CAPACITY,
 };
 use nimiq_transaction::extended_transaction::ExtTxData;
-use nimiq_transaction::{ExecutedTransaction, Transaction, TransactionFormat};
+use nimiq_transaction::{ExecutedTransaction, TransactionFormat};
 use nimiq_utils::time::OffsetTime;
 use nimiq_zkp_component::ZKPComponent;
 use parking_lot::{Mutex, RwLock};
-use std::any::Any;
 use std::str::FromStr;
 use std::sync::Arc;
```

Comment:
We should move the `std` `use`s to the beginning and separate the one for `parking_lot`